packages/AutoTDD-GUI.package/ATDDGitHubActionResults.class/instance/selectedJob..st

```diff
@@ -0,0 +1,7 @@
+accessing
+selectedJob: anObject
+	
+	selectedJob := anObject.
+	self
+		changed: #selectedJob.
+		"TODO: updateJobLog"
```

Comment:
Remove comment

Reply:
This functionality has yet to be implemented, but still needs to (fills lowest field in the Results window) so comment is justified